open-metadata-resources/open-metadata-deployment/charts/vdc-open/templates/igcproxy-deployment.yaml

```diff
@@ -0,0 +1,41 @@
+# <!-- SPDX-License-Identifier: Apache-2.0 -->
```

Comment:
is this Virtualizer related?

Reply:
Nope, this is not my file. I didn't modify it.